examples/configuration_api_example.py

```diff
@@ -0,0 +1,26 @@
+''' Configuration API client example usage. '''
+
+from livechat.configuration.client import ConfigurationApi
+
+# Get group name.
+configuration_api = ConfigurationApi.get_client(token='<your access token>')
+groups = configuration_api.list_groups()
+group_1 = groups.json()[1]['id']
+group_1_info = configuration_api.get_group(id=1)
+name = group_1_info.json()['name']
```

Comment:
In my opinion, this example is a little bit too "complex" with "magic" numbers. We should add here something simple, similar to https://developers.livechat.com/docs/management/configuration-api/#list-groups. For example how to get list of existing groups, etc.

Reply:
👍 ok, I am going to remove it and keep as following example only: Get list of existing groups.